application/F3DOptionsTools.cxx

```diff
@@ -122,7 +122,7 @@ static inline const std::array<CLIGroup, 8> CLIOptions = {{
     { {"bg-color", "", "Background color", "<R,G,B>", ""},
       {"resolution", "", "Window resolution", "<width,height>", ""},
       {"position", "", "Window position", "<x,y>", ""},
-      {"fps", "z", "Display frame per second", "<bool>", "1"},
+      {"fps", "z", "Display rendering frame per second", "<bool>", "1"},
```

Comment:
Yeah, speaking of that, it's currently not working. See https://github.com/f3d-app/f3d/issues/1745

Reply:
Yeah, ive seen that.